rescript-mode.el

```diff
@@ -169,6 +169,32 @@
   ;; Fonts
   (setq-local font-lock-defaults '(rescript--font-lock-keywords)))
 
+
+;;; Compilation Error Regexs
+(eval-and-compile
+  (defconst rescript--compilation-error-rx
+    (rx (seq
+          "We've found a bug for you!"
+          (* (or space control))
+          (group (seq (* any) ".res"))
+          ":"
+          (group (+ digit))
+          ":"
+          (group (+ digit))
+          "-"
+          (group (+ digit))
+          line-end)))
```

Comment:
There's also the possibility of syntax errors, which have a slightly different format. This covers both:

```suggestion
    (rx (seq
          (or "We've found a bug for you!" "Syntax error!")
          (* (or space control))
          (group (seq (* any) ".res"))
          ":"
          (group (+ digit))
          ":"
          (group (+ digit))
          (? "-"
             (group (+ digit)))
          line-end)))
```